### ISSUE TITLE
Bus Mouse emulation?

### ISSUE THREAD

**Issue by Siddhant-Mashalkar** [joncampbell123/dosbox-x#2972]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar question(s) already exists?

- [X] I have searched and didn't find any similar issues.

### Question

i was trying Microsoft Mouse Bundle 1.0, which only supports microsoft bus mouse, when i try to run mouse, it says `Mouse: Mouse hardware not found`.   does DOSBox-X have bus mouse emulation? if yes, how to enable it?

**Comment by rderooy** [comment]
No. By default the DOSBox-X (just like vanilla DOSBox) emulates a PS/2 mouse.

DOSBox-X (unlike vanilla DOSBox) does however support serial mouse emulation, and since the early Microsoft mouse was sold in both Bus mouse and Serial editions, you may want to try that.

In your dosbox-x config file set the following option:
```
[serial]
serial1=serialmouse
```

**Comment by Torinde** [comment]
relevant documentation and info about Bus mouse emulation in VirtualBox: https://www.pcjs.org/blog/2017/10/13/

**Comment by rderooy** [comment]
86box supports bus mouse if you really need it.
